src/utils/components/AddBootableVolumeModal/components/VolumeMetadata/components/InstanceTypeDrilldownSelect/components/ComposableDrilldownSelect/hooks/useClickOutside.ts

```diff
@@ -0,0 +1,22 @@
+import { RefObject, useEffect } from 'react';
```

Comment:
aviv already added this one - is it merged? u moved? same file?

Reply:
please use the hook in here instead: 
https://github.com/kubevirt-ui/kubevirt-plugin/blob/d800ea5b99d7fb6e5b7045a116145256062fcad8/src/utils/hooks/useClickOutside/useClickOutside.ts